bittensor/_axon/__init__.py

```diff
@@ -31,6 +30,7 @@
 import grpc
 from substrateinterface import Keypair
 
+import json
 import bittensor
```

Comment:
I know that you did not change this in this PR, but as I mentioned in other PR,
What do you think about changing this absolute import to a relative one since we are inside the `bittensor` module

Reply:
Hey, @eduardogr cause the same practise have been widely used across the project, so i think it might be better if we can change them in a separate PR.

Cause axon is a rather big submodule and it is using a lots of other bittensor submodules like prometheus /  wallet / proto / threadpool / config / grpc.

Reply:
ok, i see, fair enough